bundles/admin/admin-layereditor/view/AdminLayerForm/AdminLayerFormHandler.js

```diff
@@ -30,7 +31,8 @@ class UIHandler extends StateHandler {
             loading: false,
             tab: DEFAULT_TAB,
             credentialsCollapseOpen: false,
-            scales: mapmodule.getScaleArray().map(value => typeof value === 'string' ? parseInt(value) : value)
+            scales: mapmodule.getScaleArray().map(value => typeof value === 'string' ? parseInt(value) : value),
+            tempStyle: OSKARI_BLANK_STYLE
```

Comment:
Don't initialize style here. Instead make StyleEditor use the blank style to initialize itself merging it with provided style.